packages/pyxis-react/src/components/Form/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,222 @@
+import React, {
+  ChangeEvent,
+  FC,
+  ForwardedRef,
+  forwardRef,
+  InputHTMLAttributes,
+  ReactElement,
+  useEffect, useRef,
+  useState,
+} from 'react';
+import Dropdown, { getDropdownId } from 'internalComponents/Dropdown';
+import {
+  IconClose, IconSearch,
+} from 'components/Icons';
+import { ButtonFC } from 'components/Button/types';
+import FormField, { getFieldId } from 'internalComponents/FormField';
+import useClickOutside from 'commons/hooks/useClickOutside';
+import withFormItem, { getErrorId, getHintId } from 'hoc/withFormItem';
+import classNames from 'classnames';
+import {
+  Item, filterItems, getItemFromValue, FilteringFunction,
+} from 'internalComponents/Dropdown/items';
+import useDropdown, { initialDropdownConfig } from 'commons/hooks/useDropdown';
+import keyDownHandler from 'internalComponents/Dropdown/keyDownHandler';
+import { getItemIdFromValue, Suggestion } from 'internalComponents/Dropdown/Dropdown';
+import useCombinedRefs from 'commons/hooks/useCombinedRefs';
+
+const getClasses = (size: FieldSize, isFilled: boolean): string => classNames(
+  'form-field__autocomplete',
+  {
+    'form-field__autocomplete--filled': isFilled,
+    'form-field__autocomplete--small': size === 'small',
+  },
+);
+
+const getResetClasses = (size: FieldSize): string => classNames(
+  'form-field__addon__reset',
+  {
+    'form-field__addon__reset--small': size === 'small',
+  },
+);
+
+export const Autocomplete: FC<AutocompleteProps> = forwardRef<HTMLInputElement, AutocompleteProps>(({
+  className,
+  disabled,
+  dropdownHeader,
+  errorMessage,
+  filteringFunction = 'startsWith',
+  hasError,
+  hint,
+  id,
+  isCaseSensitive = false,
+  items,
+  name,
+  noResultFoundAction,
+  noResultFoundText,
+  onValueChange,
+  onChangeFilteringValue,
```

Comment:
We should try to keep consistency between those two attributes.

```suggestion 
onValueChange, 
onFilterValueChange,
```